subprojects/testfx-core/testfx-core.gradle

```diff
@@ -16,7 +16,8 @@
  */
 apply plugin: 'java-library'
 
-ext.pomDescription = 'TestFX Core'
+ext.pomName = 'TestFX Core'
+ext.pomDescription = 'TestFX core library'
 ext.moduleName = 'org.testfx'
 ext.openjfxVersion = '12'
```

Comment:
here is the version 12 in the other modules the version is 11 - why?

Reply:
Probably just oversight. I will set this to 11 as the others are.